.gitignore

```diff
@@ -13,4 +13,5 @@ node_modules/
 npm-debug.log
 
 .DS_Store
+.history
```

Comment:
I'm not familiar with this file - is this something that could go in your global gitignore, instead of in project-specific gitignores?